dataset_builders/pie/argmicro/README.md

```diff
@@ -0,0 +1,31 @@
+# PIE Dataset Card for "ArgMicro"
+
+This is a [PyTorch-IE](https://github.com/ChristophAlt/pytorch-ie) wrapper for the
+[ArgMicro Huggingface dataset loading script](https://huggingface.co/datasets/DFKI-SLT/argmicro).
+
+## Data Schema
+
+The document type for this dataset is `ArgMicroDocument` which defines the following data fields:
+
+- `id` (str)
+- `text` (str)
+- `topic_id` (str, optional)
+- `stance` (str, optional)
```

Comment:
This is now also an annotation layer, see https://github.com/ArneBinder/pie-datasets/pull/56/commits/52afbb66e79d2cfb23084ab98de398a60123a7d6

Reply:
edit in 8e97ed3